Sprint-1/exercises/paths.js

```diff
@@ -11,8 +11,16 @@
 
 const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
 const lastSlashIndex = filePath.lastIndexOf("/");
+    console.log(lastSlashIndex);
 const base = filePath.slice(lastSlashIndex + 1);
+    console.log(base);
 console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+
+const dirPart = filePath.substring(45, -1);
```

Comment:
Your code should be written to to work for any valid value of `filePath`.
For example, `"/path1/path2/package.json"`.